common/djangoapps/entitlements/admin.py

```diff
@@ -14,3 +15,11 @@ class EntitlementAdmin(admin.ModelAdmin):
                     'mode',
                     'enrollment_course_run',
                     'order_number')
+
+
+@admin.register(CourseEntitlementPolicy)
+class EntitlementPolicyAdmin(admin.ModelAdmin):
```

Comment:
Nit: Should this be named `CourseEntitlementPolicyAdmin`?

Reply:
Good nit, will update the above EntitlementPolicy as well for consistency